frontend/pages/beta/index.tsx

```diff
@@ -125,17 +113,19 @@ export default function ExploreModels() {
               // TODO fetch all model libraries
               tags={['PyTorch', 'TensorFlow', 'JAX', 'Transformers', 'ONNX', 'Safetensors', 'spaCy']}
               expandThreshold={10}
-              selectedTags={selectedLibrary}
-              onChange={setSelectedLibrary}
+              multiple
+              selectedTags={selectedLibraries}
+              onChange={setSelectedLibraries}
               size='small'
             />
           </Box>
           <Box>
-            <ChipSelector<MarketPlaceModelSelectType>
+            <ChipSelector
               label='Other'
-              tags={[MarketPlaceModelSelectType.MY_MODELS, MarketPlaceModelSelectType.FAVOURITES]}
-              onChange={setSelectedType}
-              selectedTags={selectedType}
+              multiple
+              tags={['mine']}
```

Comment:
Everything else starts with a capital, can we make this an object so that we can display it as 'Mine' but send the code as 'mine'?